src/main/java/commands/EventCommand.java

```diff
@@ -0,0 +1,64 @@
+package commands;
+
+import common.Parser;
+import common.Ui;
+import dukeexceptions.DukeException;
+import dukeexceptions.MissingDescriptionException;
+import dukeexceptions.WrongDatetimeFormatException;
+import tasklist.TaskList;
+import tasks.Event;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.ResolverStyle;
+
+public class EventCommand extends Command {
+    static final String EVENT_DATETIME_START_OR_END_FORMAT = "d/MM/uuuu HHmm";
+    static final String EVENT_DATETIME_START_OR_END_STORAGE_FORMAT = "MMM dd uuuu, HHmm";
+    static final String EVENT_DATETIME_INPUT_FORMAT = "d/MM/uuuu HHmm - d/MM/uuuu HHmm";
+    private final String[] args;
+
+    public EventCommand(String[] args) {
+        this.args = args;
+    }
+
+    public static void validateArguments(String[] args) throws DukeException {
+        String description = Parser.splitArrayIntoSubstrings(args, "/at").get(0);
+
+        // ERROR HANDLING: Check for empty Tasks.Deadline description
+        if (description.equalsIgnoreCase("")) {
+            throw new MissingDescriptionException("event");
+        }
+        String unparsedTimeRange = Parser.splitArrayIntoSubstrings(args, "/at").get(1);
+        String unparsedStartDateTime = unparsedTimeRange.split("-")[0].strip();
+        String unparsedEndDateTime = unparsedTimeRange.split("-")[1].strip();
+
+        if (!Parser.isValidDatetime(unparsedStartDateTime, EVENT_DATETIME_START_OR_END_FORMAT) || !Parser.isValidDatetime(unparsedEndDateTime, EVENT_DATETIME_START_OR_END_FORMAT)) {
```

Comment:
This line seems a little to long. Perhaps you can consider the following code:
```suggestion
        bool isValidStartDatetime = !Parser.isValidDatetime(unparsedStartDateTime, EVENT_DATETIME_START_OR_END_FORMAT);
        bool isValidEndDatetime = !Parser.isValidDatetime(unparsedEndDateTime, EVENT_DATETIME_START_OR_END_FORMAT);
        if (!isValidStartDatetime || !isValidEndDatetime) {
```